bin/pycbc_live

```diff
@@ -260,12 +260,20 @@ class LiveEventManager(object):
             f.attrs['command_line'] = sys.argv
             f.attrs['num_live_detectors'] = len(self.live_detectors)
 
+            def h5py_unicode_workaround(stuff):
+                # FIXME workaround for the fact that h5py cannot handle Unicode
+                # numpy arrays that show up with Python 3
+                if hasattr(stuff, 'dtype') and 'U' in stuff.dtype.str:
```

Comment:
Check if `stuff.dtype.kind == 'U'` in case any weird cases might have 'U' in the dtype name.

Reply:
:+1: